frontend/plugins/accounting_ui/src/modules/transactions/transaction-form/components/forms/InvIncomeForm/ExpenseForm.tsx

```
@@ -0,0 +1,113 @@
+
+  return (
+    <RecordTableHotkeyProvider
+      columnLength={5}
+      rowLength={fields.length}
+      scope={AccountingHotkeyScope.TransactionCEPage}
+    >
+      <Table className="mt-8 p-1 overflow-hidden rounded-lg bg-sidebar border-sidebar">
+        <InventoryTableHeader form={form} journalIndex={journalIndex} />
+        <Table.Body className="overflow-hidden">
+          {fields.map((product, detailIndex) => (
+            <></>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)